property.go

```diff
@@ -20,6 +20,33 @@ type PropertyDefinitionConnection struct {
 	TotalCount int `graphql:"-"`
 }
 
+type PropertyInput struct {
+	Owner         IdentifierInput `json:"owner"`
+	Definition    IdentifierInput `json:"definition"`
+	Value         JSONString      `json:"value"`
+	RunValidation *bool           `json:"runValidation,omitempty"`
+}
+
+// Treating Definition (PropertyDefinition) as an IdentifierInput
+// Treating Owner (... on Service) as an IdentifierInput
+type Property struct {
+	Definition       IdentifierInput  `graphql:"definition"`
+	Owner            IdentifierInput  `graphql:"owner"`
+	ValidationErrors []OpsLevelErrors `graphql:"validationErrors"`
```

Comment:
In the hackday today with foundations it looks like they actually have a new error type - we may need to look at it and see if its actually like our `OpsLevelErrors` struct or if we need to create a new type